tests/unit/Exception/HubspotExceptionTest.php

```diff
@@ -0,0 +1,79 @@
+<?php
+declare(strict_types=1);
+
+namespace SevenShores\Hubspot\Tests\unit\Exception;
+
+use GuzzleHttp\Exception\ClientException;
+use GuzzleHttp\Exception\RequestException;
+use GuzzleHttp\Exception\ServerException;
+use GuzzleHttp\Psr7\Request;
+use GuzzleHttp\Psr7\Response;
+use PHPUnit\Framework\TestCase;
+use SevenShores\Hubspot\Exceptions\BadRequest;
+use SevenShores\Hubspot\Exceptions\HubspotException;
+use function GuzzleHttp\Psr7\stream_for;
+
+class HubspotExceptionTest extends TestCase
+{
+    const EXAMPLE_TOKEN = '8907e60c-600d-4af8-a987-191c104a215c';
+
+    public function testCreateExceptionFromGuzzleRequestException()
```

Comment:
it would be better  if you mark all test methods  /** @test */ 
and Could you use static it would be better

Reply:
Sorry not sure what you mean by this?
The annotation is only required if the test method doesn't start with `test` (which they do)

Reply:
Test methods should not be static either
https://phpunit.readthedocs.io/en/8.4/writing-tests-for-phpunit.html

Reply:
Sorry for confusion
I'll describe it

Reply:
Regarding annotation
All our tests marked  as /** @test */
to follow project style please

Reply:
Updated